### PR TITLE
EventBridge API fixes

### PR DESCRIPTION
**Pull request by zachmargolis**
Discovered from testing in `dev`

### CODE REVIEW

File: app/services/risc_destination_updater.rb [hunk 1]
```diff
@@ -24,27 +24,36 @@ def remove
     eventbridge_client.delete_connection(name: connection_name) if connection_exists?
   end
 
+  def issuer_slug
+    service_provider.issuer.gsub(/[^.\-_A-Za-z0-9]/, '_')
```
Comment (stevegsa) [inline app/services/risc_destination_updater.rb:28]: perhaps some memoization?

**Review by stevegsa — APPROVED** [review]
lgtm

File: app/services/risc_destination_updater.rb (hunk 1 above)
Comment (aduth) [inline app/services/risc_destination_updater.rb:28]: Could use [`ActiveSupport::Inflector.parameterize`](https://api.rubyonrails.org/v6.1.3.2/classes/ActiveSupport/Inflector.html#method-i-parameterize) here?

```suggestion
    ActiveSupport::Inflector.parameterize(service_provider.issuer, separator: '_')
```

Or if we'd be fine with dashes as the separator:

```suggestion
    service_provider.issuer.parameterize
```

File: app/services/risc_destination_updater.rb [hunk 2]
```diff
@@ -24,27 +24,36 @@ def remove
...
   end
 
   def event_bus_name
     # Matches value managed by Terraform
     "#{Identity::Hostdata.env}-risc-notifications"
   end
 
+  def destination_iam_role_arn
+    # Matches value managed by Terraform
+    "arn:aws:iam::#{aws_account_id}:role/#{Identity::Hostdata.env}-risc-notification-destination"
```
Comment (aduth) [inline app/services/risc_destination_updater.rb:54]: Any way we could incorporate a test assertion here? Partly also wondering if `::` is expected?

**Review by aduth — APPROVED** [review]

File: app/services/risc_destination_updater.rb (hunk 1 above)
Comment (zachmargolis) [reply app/services/risc_destination_updater.rb:28]: This regex came from inverting the regex in error message I got from the API through trial and error, so I figure it's easier to maintain as-is?

File: app/services/risc_destination_updater.rb (hunk 2 above)
Comment (zachmargolis) [reply app/services/risc_destination_updater.rb:54]: so it turns out... the `::` is not expected it was a copy-paste error on my part, but yes will add a test case

File: app/services/risc_destination_updater.rb (hunk 1 above)
Comment (zachmargolis) [reply app/services/risc_destination_updater.rb]: done in c4fa91c00ef46d11e4ee9b63bd5eb16c1a9b765e

File: app/services/risc_destination_updater.rb (hunk 2 above)
Comment (zachmargolis) [reply app/services/risc_destination_updater.rb:54]: I got some ARNs confused... I believe this *should* have the `::`... added a spec in 89763b0fa39693d2dafb7fd76f1a897fa2e733f8, [slack message why i think this is right](https://gsa-tts.slack.com/archives/C01D7MWNE64/p1624397530017100?thread_ts=1624374211.009400&cid=C01D7MWNE64)

**Pull request merged**